test/testinfra.py

```diff
@@ -343,11 +367,18 @@ def scan(self, update, context, except_context=False):
         results = []
         master_contexts = []
         pull_contexts = []
+        verify_contexts = []
+        image_contexts = []
         for (context, what) in contexts.items():
```

Comment:
Also this function gets called every minute, are we sure that's how often we want to scan for creating new images? Or is there a better way to do that? Am I misunderstanding how this works?